utils/CONSTANTS.py

```diff
@@ -1,45 +0,0 @@
-# Supported Background. Can add/remove background video here....
```

Comment:
Could you find a way to reinclude this

Reply:
@JasonLovesDoggo I added `"__comment": "Supported Backgrounds. Can add/remove background video here...",` at the top of the `backgrounds.json` file. I can't think of any other solution to include it in the JSON file.